### ISSUE TITLE
A11y_Fabric Extensions for VS Code_Fabric_GraphQI ApI _CCA:keyboard focus for GraphQI ApI is having color contrast ratio 1.243:1 which is less than 3:1.

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#39774]
Does this issue occur when all extensions are disabled?: No

- VS Code Version: 1.86.2
- OS Version: Windows 11 Enterprise

**Repro Steps:**
1. Launch Visual Studio Code
2. Install the "Fabric workspaces" and "Fabric App Development" extensions
3. Tab to fabric workspaces icon in left activity bar and press enter
4. Sign in to fabric with appropriate credentials and
5. Tab to command palette and type Fabric create item and
6. Select Graph QI Api preview  and type Graph QI Api
7. Tab to tree view Graph QI Api
8. Tab using keyboard 
9. Turn on the CCA tool.
10. Observe the keyboard focus ratio.

**Actual Result:**
 keyboard focus for Graph QI Api is having less Color contrast ratio.

**Note :**
Similar - has been observed for Entire screen.

**Expected Result:**
Keyboard focus for Graph QI Api should have more Color contrast ratio.

**User Impact:**
Low Vision users will face difficulties keyboard focus for Graph QI Api is having less Color contrast ratio.

**Recommendations:**
Refer [Developer Resources (sharepoint.com)](https://microsoft.sharepoint.com/sites/accessibility/SitePages/Developer-Resources.aspx?xsdata=MDV8MDF8fGY4MTA5OTIwZTkyNDQ0YzM1OTRmMDhkYmNlZDdlZDRmfDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDB8MHw2MzgzMzExOTg5OTc3NDgwNDd8VW5rbm93bnxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaVYybHVNeklpTENKQlRpSTZJazkwYUdWeUlpd2lWMVFpT2pFeGZRPT18MXxMMk5vWVhSekx6RTVPall4TWpJelkyRmxPV1pqTURRd05tUTRNemxtT0RZd05UTTVORFl3WkRNNFFIUm9jbVZoWkM1Mk1pOXRaWE56WVdkbGN5OHhOamszTlRJek1EazVNRFk1fGNlZWU4YTA0ZjkyMjQ0MTc1OTRmMDhkYmNlZDdlZDRmfGY5ODc4NWE0MTA3YTRlYWY4OTU1ZWNhNjIzOWJjMWY2&sdata=VzQyNGxUQVYwMTZnL1d0aGt3dEhWbDhIaU5oVE5hYllLYVdETyt3UXkxdz0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-harugula%40microsoft.com&OR=Teams-HL&CT=1697523148200&clickparams=eyJBcHBOYW1lIjoiVGVhbXMtRGVza3RvcCIsIkFwcFZlcnNpb24iOiIyNy8yMzA5MTQxNzUwNSIsIkhhc0ZlZGVyYXRlZFVzZXIiOmZhbHNlfQ%3D%3D)

**MAS Reference:**
[MAS 1.4.11 – Non-text Contrast](https://aka.ms/MAS1.4.11)
